web/packages/design/src/ButtonLink/ButtonLink.jsx

```diff
@@ -46,6 +46,7 @@ const StyledButtonLink = styled.a`
   &:hover,
   &:focus {
     color: ${({ theme }) => theme.colors.buttons.link.hover};
+    box-shadow: none;
```

Comment:
I'm not going to change it right now, because it breaks a bit in places where link buttons are placed on a red background (the text turns illegible). As this coincides with your another comment, I'll fix it properly after I deal with the alert banners.